docs/aeon_integration.md

```diff
@@ -0,0 +1,42 @@
+# Aeon Integration
+
+To allow users to request materials, Pulfalight submits a form to Aeon's
+external request endpoint. This triggers an aeon login and the user can then manage their requests in the Aeon UI.
+
+* [Aeon Documentation](https://support.atlas-sys.com/hc/en-us/articles/360011820054-External-Request-Endpoint). Note this documentation is not comprehensive, and a lot of what we do has been discovered through experience.
+* [Technical Overview of EAD Requesting with Aeon](https://blog.rockarch.org/technical-overview-of-ead-requesting-with-aeon)
+
+### The Aeon External Request Endpoint
+
+The form has a few different types of fields: global fields and per-request
+fields. Some of the global fields are Grouping Options.
+
+Grouping Options allows you to define a grouping field (ours is the call number
+and the box number) so that you can combine multiple requests into a single call
+slip. For each of the other values you can then define whether you want them
+concatenated or to just take the first value. Update [aeon_request.rb](https://github.com/pulibrary/pulfalight/blob/0cfda015d169ffdc34ddfeac81934382a147b307/app/values/aeon_request.rb#L174-L189) to change this logic.
+
+The other global fields we don't really because we submit multiple requests at
```

Comment:
Turns out this isn't a sentence. Also I'm not sure it's entirely true. Maybe we should say something like "Otherwise we mostly prefer per-request fields over global fields because we submit multiple requests at once."